packages/jest-runtime/src/index.ts

```diff
@@ -525,6 +525,48 @@ export default class Runtime {
               // @ts-expect-error Jest uses @types/node@16. Will be fixed when updated to @types/node@20.11.0
               meta.dirname = path.dirname(meta.filename);
 
+              meta.resolve = async (
```

Comment:
can this use `this._resolveModule`? then it should be enough to just handle the URL bit, and not have to care about `this._resolver` at all

Reply:
I changed it to do so now